viewer/packages/rendering/src/glsl/sector/mesh.frag

```diff
@@ -36,7 +37,9 @@ void main()
     }
 
 #if defined(IS_TEXTURED)
-    vec3 diffuseColor = texture(tDiffuse, v_uv).rgb;
+    // Gamma correction of 1.3 times to match GLTFViewer (Magic number determined based on visual comparison).
+    vec3 diffuseColor = pow(texture(tDiffuse, v_uv).rgb, vec3(1.3));
+    diffuseColor = LinearTosRGB(vec4(diffuseColor, 1.0)).xyz;
```

Comment:
I guess this is fine - although I don't mind having a bit of difference from the gltf-viewer. Would be nice to know what they are doing differently from us though :thinking:

Reply:
Here I've put the original image texture to the right of the screenshot above: 
![bg1](https://user-images.githubusercontent.com/70905152/224251411-707769a9-1769-45e2-9dff-ea14e2078fae.png)
I think it's more than close enough for us to accept it